README.md

```diff
@@ -49,90 +49,79 @@ Example badges for `mio` crate:
 
 ## Development
 
-We strongly recommend using vagrant, this will give you a virtual machine
-already configured and ready to start developing on.
+We strongly recommend using docker-compose, which will make it easier to get started
+without adding new users and packages to your host machine.
 
 ### Getting started
 
-Make sure you have vagrant, virtualbox and a ssh client and you need
-to able to download ~800MB data on the first run.
+Make sure you have docker-compose and are able to download ~10GB data on the first run.
 
 
 ```sh
 git clone https://github.com/rust-lang/docs.rs.git docs.rs
 cd docs.rs
-vagrant up  # This may take a little while on the first run
+docker-compose up  # This may take a half hour or more on the first run
 ```
 
-You can always run `vagrant provision` to reconfigure virtual machine.
-Provision will install required dependencies and nightly rust compiler
-into virtual machine. It will also configure lxc-container inside
-virtual machine.
-
 ### CLI
 
-Make sure you are running every listed command inside `/vagrant` directory
-in virtual machine. You can connect to virtual machine with `vagrant ssh` and
-switch current working directory with: `cd /vagrant` inside virtual machine.
-
+Make sure you are running every listed command prefixed with
+`docker-compose run web`. If you want to run the web server, you will also need
+the flag `-p 3000:3000` to allow connections to the container.
 
 #### Starting web server
 
 This command will start web interface of docs.rs and you can access it from:
-`http://localhost:3000/`
-
-```
-cargo run -- start-web-server
-```
-
+`http://localhost:3000/`: `start-web-server`
 
 #### `build` subcommand
 
 ```sh
 # Builds <CRATE_NAME> <CRATE_VERSION> and adds it into database
 # This is the main command to build and add a documentation into docs.rs.
-cargo run -- build crate <CRATE_NAME> <CRATE_VERSION>
+# For example, `docker-compose run web build crate regex 1.1.6`
+build crate <CRATE_NAME> <CRATE_VERSION>
```

Comment:
Maybe prefix the commands with `docker-compose run web`? I'd love for them to be copy-pastable.